contracts/upgradeable_contracts/HomeOmnibridge.sol

```diff
@@ -98,18 +129,6 @@ contract HomeOmnibridge is BasicOmnibridge, HomeOmnibridgeFeeManager, MultiToken
         return nativeTokenAddress(_homeToken);
     }
 
-    /**
-     * @dev One-time upgrade function for transferring ownership of the STAKE token to the TokenMinter address.
-     * Should be called together with upgradeToAndCall function
-     */
-    function transferTokenOwnership() external {
```

Comment:
Don't you mind to create a separate PR to remove this method?

Reply:
Sure.